app/src/main/webapp/WEB-INF/jsp/tiles/views/home/search.jsp

```diff
@@ -1,5 +1,7 @@
 <%--@elvariable id="resourcesVersion" type="String"--%>
 <%--@elvariable id="geneId" type="String"--%>
+<%--@elvariable id="resourcesVersion" type="String"--%>
```

Comment:
Are lines 3 and 4 the duplicate of lines 1 and 2?
If yes, then could we remove them, please?